lib/foodcritic/command_line.rb

```diff
@@ -76,7 +77,8 @@ def initialize(args)
           @options[:version] = true
         end
         opts.on('-X', '--exclude PATH',
-                'Exclude path(s) from being linted.') do |e|
+                'Exclude path(s) from being linted. Default test/**/*,'\
+                'spec/**/*,features/**/*') do |e|
```

Comment:
Unfortunately in it's current state you'd have to pass some bogus dir via the CLI to override this.  Something like -X none would do it. That's less than ideal and I'm open up to a better solution.